### PR TITLE
Developer Call Notes - 2020-11-12

### PR DESCRIPTION
**Pull request by riedgar-ms**
Note from the developer call.

Unless there is feedback, will merge on Monday.

### CODE REVIEW

File: docs/contributor_guide/developer_call_notes/2020_11_12.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Developer Call 2020-10-08
```
Comment (romanlutz) [inline docs/contributor_guide/developer_call_notes/2020_11_12.rst:1]: Please update the date. Also, the previous one didn't have the same format as all previous ones (which spelled out the month). Please unify.

**Review by romanlutz — APPROVED** [review]
Thanks for writing this, @riedgar-ms ! Really helpful whenever one can't make it :-)

File: docs/contributor_guide/developer_call_notes/2020_11_12.rst (hunk 1 above)
Comment (riedgar-ms) [reply docs/contributor_guide/developer_call_notes/2020_11_12.rst]: Fixed the date; is it worth converting all the others to ISO format?

Comment (romanlutz) [reply docs/contributor_guide/developer_call_notes/2020_11_12.rst]: It's all manually written. There's no automatic conversion. That's why we should change this and the previous one as well:

image

Comment (romanlutz) [reply docs/contributor_guide/developer_call_notes/2020_11_12.rst]: @riedgar-ms  I think you wrote "<n>th <month>" now as opposed to "<month> <n>". I'm fine either way, but let's be consistent with all of them (including last month's). So you can either make all of them conform to your way of writing it or adjust the last two to the existing way.

Comment (romanlutz) [reply docs/contributor_guide/developer_call_notes/2020_11_12.rst]: I'm afraid the ordering is still different. Yours says 8 October 2020, for example, whereas the ones above (in the screenshot) say September 17, 2020. Please just get them all aligned in whichever way you prefer.

**Comment by romanlutz** [comment]
Thanks! Feel free to merge since this is really a trivial change.

**Pull request merged**